operatorconfig/driverconfig/unity/v2.11.0/upgrade-path.yaml

```diff
@@ -0,0 +1 @@
+minUpgradePath: v2.9.1
```

Comment:
@shefali-malhotra Value here should be v2.10.0.. can you please check

Reply:
addressed